bindings/python/Cargo.toml

```diff
@@ -13,21 +13,20 @@ rayon = "1.3"
 serde = { version = "1.0", features = [ "rc", "derive" ]}
 serde_json = "1.0"
 libc = "0.2"
-env_logger = "0.7.1"
-pyo3 = "0.12"
-numpy = "0.12"
-ndarray = "0.13"
+env_logger = "0.9.0"
+pyo3 = "0.16.2"
+numpy = "0.16.2"
+ndarray = "0.15"
 onig = { version = "6.0", default-features = false }
-itertools = "0.9"
+itertools = "0.10"
 
 [dependencies.tokenizers]
 version = "*"
 path = "../../tokenizers"
 
 [dev-dependencies]
 tempfile = "3.1"
+pyo3 = { version = "0.16.2", features = ["auto-initialize"] }
```

Comment:
`auto-initialize` feature is required for `cargo test` to work.

Reply:
I think adding it here without also adding `resolver = "2"` in a leaf crate (this will build a cdylib at the end of the day) means that the features will be unified, so one could just as well enable the features on the dependency itself. (But I think adding `resolver = "2"` would be nicer in any case, but yeah another change in an already big PR.)

Reply:
CI was running with `cargo test --no-default-features` so the `extension-module`  feature is disabled thus only `auto-initialize` is enabled.

Reply:
Yes, certainly. I just mean that having it as a separate `dev-dependency` instead of enabling the feature on the original `dependency` does only help if `resolver = "2"` is enable. Otherwise, the activated features will be merge over dependencies and development dependencies.

Reply:
Right, I can add `resolver = "2"` if the newer Rust version requirement (1.51.0) is acceptable.